types/justified-layout/index.d.ts

```diff
@@ -0,0 +1,127 @@
+     * nicely. The algorithm will get as close to the target row height as it can.
+     * @default 320
+     */
+    targetRowHeight?: number;
+    /**
+     * How far row heights can stray from targetRowHeight. `0` would force rows to be the
+     * `targetRowHeight` exactly and would likely make it impossible to justify. The value must
+     * be between `0` and `1`.
+     * @default 0.25
+     */
+    targetRowHeightTolerance?: number;
+    /**
+     * Will stop adding rows at this number regardless of how many items still need to be laid
+     * out.
+     * @default Number.POSITIVE_INFINITY
+     */
+    maxNumRows?: number;
+    /**
+     * Provide an aspect ratio here to return everything in that aspect ratio. Makes the values
+     * in your input array irrelevant. The length of the array remains relevant.
+     * @default false
+     */
+    forceAspectRatio?: boolean | number;
+    /**
+     * If you'd like to insert a full width box every n rows you can specify it with this
+     * parameter. The box on that row will ignore the targetRowHeight, make itself as wide as
+     * `containerWidth - containerPadding` and be as tall as its aspect ratio defines. It'll
+     * only happen if that item has an aspect ratio >= 1. Best to have a look at the examples to
+     * see what this does.
+     * @default false
+     */
+    fullWidthBreakoutRowCadence?: boolean | number;
+    /**
+     * By default we'll return items at the end of a justified layout even if they don't make a
+     * full row. If false they'll be omitted from the output.
+     * @default true
+     */
+    showWidows?: boolean;
+    /**
+     * If widows are visible, how should they be laid out?
+     * @default "left"
+     */
+    widowLayoutStyle?: 'left' | 'justify' | 'center';
+}
+
+/**
+ * Computed positional and sizing properties of a box in the layout.
+ */
+export interface LayoutBox {
+    /**
+     * Aspect ratio of the box.
+     */
+    aspectRatio: number;
+    /**
+     * Distance between the top side of the box and the top boundary of the justified layout.
+     */
+    top: number;
+    /**
+     * Width of the box in a justified layout.
+     */
+    width: number;
+    /**
+     * Height of the box in a justified layout.
+     */
+    height: number;
+    /**
+     * Distance between the left side of the box and the left boundary of the justified layout.
+     */
+    left: number;
+}
+
+/**
+ * Results from calculating the justified layout.
+ */
+export interface JustifiedLayoutResult {
+    /**
+     * Height of the container containing the justified layout.
+     */
+    containerHeight: number;
+    /**
+     * Number of items that are in rows that aren't fully-packed.
+     */
+    widowCount: number;
+    /**
+     * Computed positional and sizing properties of a box in the justified layout.
+     */
+    boxes: LayoutBox[];
+}
+
+export default function createJustifiedLayout(
```

Comment:
This package doesn't have a `default` property, as [tested on runkit](https://npm.runkit.com/justified-layout):

```
var justifiedLayout = require("justified-layout")
console.log(justifiedLayout.default)
```

Logs `undefined`. The correct syntax is `export = createJustifiedLayout;` (and associated `import createJustifiedLayout = require('justified-layout')`)

Reply:
Thank you for taking the time to review! I've made the changes.